wavefileexporter.cpp

```diff
@@ -129,14 +129,16 @@ void WaveFileExporter::writeSoundAt(unsigned int timestamp)
         outputData << (qint8)0;
     outputFile.seek(beatStartSample + SIZE_OF_SIMPLE_WAV_HEADER);
     inputFile.seek(inputDataChunkStartPos + 8);
-    char frame[bytesPerFrame];
+    char* frame = new char[bytesPerFrame];
```

Comment:
The reason for this change is that this code results in a compile error on msvc because it does not support VLA's, and this seemed like the best solution without getting too involved with the project for the time being. I ran both your released and the locally compiled version, but there are quite a few things that don't seem to work. For example, on Windows, you need to install the k-lite codec pack for videos to play, but even then the video didn't appear, and I had to untick and tick some beat setting (which I can't remember the name of right now). 

Reply:
Ah, OK, understood. Good old Microsoft making life slightly trickier in this case (though I suppose they're not obliged to implement a C99 feature). Happy to pull it through as-is.

Thanks for the feedback on usage. I've not had problems on the machine where I develop it, but it's far from a clean machine (still running windows 10), so clearly not the experience most newcomers will get. I think I remember there are some hoops to jump though with Qt multimedia sometimes.
If I haven't mentioned it, I'll update the readme to mention the k-lite pack (and if I get the time and inclination, I'll do a bit of googling for a better solution - surely every Qt-based multimedia app can't require the user to also install k-lite?).
There is some weirdness where the video will sometimes work better if you enter and then leave fullscreen again. Sounds more like I've initialised something wrong than Qt's fault, but I remember spending some time trying differnt things with no success.